packages/elements/src/chart/index.ts

```diff
@@ -113,40 +73,18 @@ export class Chart extends BasicElement {
    * Required properties, needed for chart to work correctly.
    * @returns config
    */
-  protected get requiredConfig (): ChartConfig {
+  protected get requiredConfig (): ChartOptions {
     return {
-      options: {
-        responsive: false, // use framework resize events instead
-        maintainAspectRatio: false,
+      responsive: true,
+      maintainAspectRatio: false,
+      plugins: {
         title: {
           display: false
         }
       }
     };
   }
 
-  /**
-   * Safely returns the chart title
-   * @returns chart title
-   */
-  protected get chartTitle (): string {
-    const title = this.config?.options?.title?.text;
-
-    if (title) {
-      return typeof title === 'string' ? title : title.join();
-    }
-
-    return '';
-  }
-
-  /**
-   * Safely returns a dataset array
-   * @returns dataset array
-   */
-  protected get datasets (): Chart.ChartDataSets[] {
```

Comment:
we don't need this API anymore?

Reply:
I think it hasn't been used yet. So I deleted it first.